modules/couchbase/couchbase.go

```diff
@@ -0,0 +1,103 @@
+package couchbase
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/netdata/go.d.plugin/pkg/web"
+)
+
+func init() {
+	module.Register("couchbase", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery: 10,
```

Comment:
Why default `UpdateEvery` is 10?

Reply:
> The REST APIs should be polled minutely via a local agent or remotely using the node(s) IP or hostname.
Couchbase REST APIs must be accessed using administrative account credentials; a Read-Only
Administrator is recommended for this purpose.
As most of the metrics provided by the REST API are per-node, it is necessary to query every node in the
cluster.
Limit the number of requests per API when querying metrics, i.e. return all bucket metrics in one request
rather than issuing separate requests per metric. Heavy use of the Couchbase REST APIs can have CPU
utilization impacts on the cluster.

I thought maybe 10s would be okay due to the comment, but I see for elastic is 5s. what do you think it should be? 5s?

Reply:
> The REST APIs should be polled minutely via a local agent or remotely using the node(s) IP or hostname.

Oh, i see, lets leave it 10 seconds then.